mlflow/models/evaluation/base.py

```diff
@@ -161,6 +332,7 @@ def make_metric(
     metric_details=None,
     metric_metadata=None,
     genai_metric_args=None,
+    require_strict_signature=False,
```

Comment:
can we define a private function to hide this argument?

```
def _make_metric(..., require_strict_signature)
    ...

def make_metric(...):
    return _make_metric(..., require_strict_signature=False)
```

Reply:
Emm but you still need to pass require_strict_signature from `make_metric` to `_make_metric` right?

Reply:
Yes, but you can hide it from users.